hooks/boot/13-tuning.rb

```diff
@@ -1,7 +1,7 @@
 if has_custom_fact?('tuning')
   # TODO: automatically get the tuning sizes - standard doesn't exist and only
   # loads base. The rest maps to config/foreman.hiera/tuning/sizes/$size.yaml
```

Comment:
There is no specific down tuning file? Perhaps that should be included in the comment.

Reply:
`down tuning file` ? what does this mean?

Reply:
Perhaps I'm inventing things, but in my mind you can tune up (for larger systems) or tune down (for smaller systems). Not saying this is what we should use, but as an example:
https://github.com/theforeman/puppet-puppet/blob/9421bd4ec9fb819fb40c15789886f7bf55b12fab/spec/acceptance/hieradata/common.yaml#L5-L8

Reply:
You can change tuning profiles upward or downward if that is what you are asking? Not sure I see how it fits into this comment.

Reply:
I'll be more concrete: it states:
> The rest maps to config/foreman.hiera/tuning/sizes/$size.yaml

Which is this directory: https://github.com/theforeman/foreman-installer/tree/develop/config/foreman.hiera/tuning/sizes

There I don't see a development.yaml. The comment basically describes how `TUNING_SIZES` is populated. If we're now saying `development` exists, but doesn't have a matching file then the comment doesn't reflect reality. I'd like it to match though.

Does that help?

Reply:
Can I just get rid of this TODO?

Reply:
We could drop the TODO, but it could be helpful to understand where the values come from. You could also say: `TUNING_SIZES` should be equal to `TUNING_SIZES.keys` from `hooks/pre_commit/13-tuning.rb`. Then in the pre_commit hook you can point to `config/foreman.hiera/tuning/sizes/$size.yaml`.

Reply:
Let me know if that new wording helps

Reply:
I still think a hint that it should map to `hooks/pre_commit/13-tuning.rb` could be useful. And we will have a TODO if we want to add tuning sizes to a scenario different from katello: it'll have different requirements. Foreman by default doesn't require 20 GB RAM.